pkg/cli/admin/etcd/listmembers.go

```diff
@@ -0,0 +1,62 @@
+package etcd
+
+import (
+	"context"
+	"fmt"
+	"io"
+
+	"github.com/spf13/cobra"
+
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"k8s.io/client-go/rest"
+	kcmdutil "k8s.io/kubectl/pkg/cmd/util"
+)
+
+type ListMembersOptions struct {
+	config *rest.Config
+
+	genericclioptions.IOStreams
+}
+
+func NewCommandListMembers(f kcmdutil.Factory, streams genericclioptions.IOStreams) *cobra.Command {
+	o := ListMembersOptions{
+		IOStreams: streams,
+	}
+	cmd := &cobra.Command{
+		Use:   "list-members",
```

Comment:
do we want to align the naming with etcd client command? IE MemberList? I think it's fine to align with k8s naming standards if you feel that is more appropriate. But adm should not have to rethink common etcd commands.

Reply:
I would defer to @soltysh on this question.